### PR TITLE
Language fixes due to updates

### LINKED ISSUE #1026: Focused Review of Help Text within Program Overview
Target areas which may interrupt flow / data entry integrity + evaluate feedback from usability testing for the Program Overview portion of the App

### This task is done when...
 - [ ] criteria 1
 - [ ] criteria 2
 
 ** Don't forget to fill out the LABEL and MILESTONE before submitting. If the issue is for the CURRENT SPRINT, add it to the PROJECT. If the issue is a BUG, FUTURE ENHANCEMENT, or OUTSIDE OF THE CURRENT SPRINT, do not add it to the PROJECT (then delete this line) **

### PR DESCRIPTION
**Pull request by NAretakis**
This PR fixes lots language things throughout the app, and closes #1026 

### This pull request changes...
- fixes language in the key personnel section to better say what we mean
- fixes in-house costs references to %FTE because we don't do that anymore
- fixes program overview section with minor edits and gets reminders to show up correctly
- fixes assurance and compliance language to try and provide some clarity for users
- fixes executive summary section language to properly put helptext in each exec summary subsection
- fixes previous activities helptext and reminder display information
- fixes proposed budget helptext to display across each subsection
- added schedule summary helptext that we didn't have before

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Comment by NAretakis** [comment]
@jameshupp this one got a lot larger than I intended. Up for a review of the language and an effort to make it better?

File: web/src/i18n/locales/en/activities/overview.yaml [hunk 1]
```diff
@@ -44,5 +45,5 @@ alternatives:
     If substantive changes have been made, include statement describing alternative
     considerations made before selecting implementation strategy.
   reminder: >-
-    If not, including statement of alternative considerations from previous versions
-    of IAPD or SMHP.
+    If substantive changes have not been made, include a statement of alternative considerations from previous version
```
Comment (jameshupp) [inline web/src/i18n/locales/en/activities/overview.yaml]:
```suggestion
    If substantive changes have not been made, include a statement of alternative considerations from the previous version
```
the? a? Feels like we need an article there.

**Comment by NAretakis** [comment]
@jameshupp I think this takes care of #1026 right? If so, I'll update this into the body of the issue.

**Comment by NAretakis** [comment]
@hbillings I think this one is ready for your look. I can't quite recall if there was other helptext stuff you wanted to integrate, or maybe just wanted to use this as a jumping off point.

File: web/src/i18n/locales/en/activities/overview.yaml (hunk 1 above)
Comment (NAretakis) [reply web/src/i18n/locales/en/activities/overview.yaml]: added "the" good catch.

**Comment by hbillings** [comment]
@NAretakis I think I'll keep all the help text/body copy updates over in #1060, just to keep that chunk of work together! This looks like it should play nice with that one. Thanks for flagging this for me. :)

**Pull request merged**